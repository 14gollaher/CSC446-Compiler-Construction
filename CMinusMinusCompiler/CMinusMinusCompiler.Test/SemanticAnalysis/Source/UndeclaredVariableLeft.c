int main()
{
	z = 4.4;
}
/*This will be omitted!!!*/
