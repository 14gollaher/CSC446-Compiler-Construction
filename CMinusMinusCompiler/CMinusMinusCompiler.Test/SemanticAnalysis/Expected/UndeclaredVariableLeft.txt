ERROR: Line 3 Use of undeclared variable z
** Symbol Table at Depth 2 **
Lexeme          Class        Depth     Offset    Type         Size      Value(r)
--------------------------------------------------------------------------------


** Symbol Table at Depth 1 **
Lexeme          Class        Depth     Offset    Type         Size      Value(r)
--------------------------------------------------------------------------------
main            Function     1         -         IntToken     0         -


Completed processing UndeclaredVariableLeft.c
